readme.md

```diff
@@ -1,5 +1,39 @@
-# applib
-Cross-platform JavaScript library for Wikimedia apps
+# Library for common Javascript transforms used by both the Android and iOS Wikipedia apps
```

Comment:
I believe the "s" in [JavaScript](https://en.wikipedia.org/wiki/JavaScript) is always capitalized. Please correct me if I'm mistaken!